client/src/sections/Login/Login.tsx

```diff
@@ -21,11 +31,12 @@ export default function Login() {
         </div>
         <Card sectioned title='Please choose a login option:'>
           <ButtonGroup fullWidth>
-            <Button primary onClick={handleLogin}>Login with Google</Button>
-            <Button primary onClick={handleLogin}>Login with Facebook</Button>
+            <Button primary onClick={handleChange}>Login</Button>
           </ButtonGroup>
         </Card>
       </Card>
+      <LoginModal email={email} password={password} active={active} handleChange={handleChange}
```

Comment:
Couple of things here:
* I don't really like us using modals here, we should try to avoid them as much as possible since they require users to perform an extra click. Use modals in situations where we want to hide the modal's information by default to save space on the screen for other things. In this case, we don't really need to save space. I recommend us to use the Polaris `Form` instead.
* When you render a component with this many props, write it out like this:
```
<LoginModal
  email={email}
  password={password}
  active={active}
  handleChange={handleChange}
  ..etc
/>
```

Reply:
the reason why I used a Modal over a Form is because I am anticipating in the near future we will most likely need a registration feature as well, and if they were static forms on the login page, it would be hard to switch between the different inputs needed for login and register. Also if we do incorporate 3rd party signins, those usually appear as modals and so will align well with our current modal.

Reply:
I'm not sure I understand your first argument. Why would it be hard to switch between different inputs needed for login and register?

Your second argument makes sense. However, in this case I would argue if we even need an email and password? In fact, we probably would benefit more by spending this time researching/implementing that 3rd party signin integration than doing this (since we already have those buttons to trigger the modals).

Reply:
well apps usually have both 3rd party and standard login, cant expect everyone to have fb or google, etc... why dont we just get back to this next sat then

Reply:
Let's discuss this on Thursday.